datasets/str_sa.py

```diff
@@ -0,0 +1,80 @@
+from os import listdir
+from os.path import isfile
+
+from chipbase import Chip
+from chipbase import ChipBase
+
+
+class STR_SA(ChipBase):
+
+    def __init__(self, directory, dataset_name="STR_SA", *args, **kwargs):
+        super().__init__(dataset_name, args, kwargs)
+
+        # Get some chips
+        self.__load_chips(directory)
+
+    def __load_chips(self, directory):
+        self.chips = {}
+
+        for file in listdir(directory):
+            path = directory + '/' + file
+
+            # Only interested in certain files
+            is_valid = isfile(path)
+            is_png = path.endswith(".png")
+            is_mask = "mask" in path
+            if not is_valid or not is_png or is_mask:
+                continue
+
+            # Set all Chip variables
+            car_id = get_sa_car_id(path)
+            cam_id = get_sa_cam_id(path)
+            chip_id = str(car_id) + "_" + str(cam_id)
```

Comment:
I think it's important to have a unique key (and have it be reproducible). I think that structure is fine assume cars only go by a camera once